src/goto-analyzer/goto_analyzer_parse_options.cpp

```diff
@@ -295,6 +296,34 @@ void goto_analyzer_parse_optionst::get_command_line_options(optionst &options)
         "call-stack-recursion-limit", cmdline.get_value("call-stack"));
       options.set_option("history set", true);
     }
+    else if(cmdline.isset("loop-unwind"))
+    {
+      options.set_option("local-control-flow-history", true);
```

Comment:
The options that are set are internal only and are intended to make the handling of these options simpler.  I am reluctant to print anything here because it is nothing that the user can't set by themself.